File: backend/internal/models/Game.go
package models

import "time"

type Game struct {
	GameId     string    `json:"game_id"`
	Date       time.Time `json:"date"`
	HomeTeam   string    `json:"home_team"`
	HomeTeamId int       `json:"home_id"`
	AwayTeamId int       `json:"away_id"`
	AwayTeam   string    `json:"away_team"`
	HomeScore  int       `json:"home_score"`
	AwayScore  int       `json:"away_score"`
}

File: frontend/src/models/game.ts
export interface Game {
    game_id: string;
    date: string;
    home_team: string;
    home_team_id: string;
    away_team: string;
    away_team_id: string;
    home_score: number | null;
    away_score: number | null;
    status: string;
    winner: string | null;
}